source/installation_guide/upgrading/from-2.3-to-3.0.rst

```diff
@@ -161,6 +161,9 @@ Removed features and their replacements
 |                                                            | default. It can be disabled by ``no-cleanup-uncertain`` - see                            |
 |                                                            | :ref:`dictmap_configuration_parameters` for more details.                                |
 +------------------------------------------------------------+------------------------------------------------------------------------------------------+
+| ``replicator``                                             | Use NFS or some other shared filesystem instead.                                         |
```

Comment:
I think it would be better to:
- have a link to a relevant NFS page, and
- "or a different" instead of "or some other" (the latter might read a bit pejorative).